.k8s/prod/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: laa-court-data-ui-app-ingress
+  namespace: laa-court-data-ui-prod
+spec:
+  rules:
+    - host: prod.view-court-data.service.justice.gov.uk
+      http:
+        paths:
+          - path: /
+            backend:
+              serviceName: laa-court-data-ui-app-service
+              servicePort: 80
+  tls:
+    - hosts:
+      - prod.view-court-data.service.justice.gov.uk
```

Comment:
this should just be `view-court-data.service.justice.gov.uk`

Reply:
👍